src/cluster_slot_stats.c

```diff
@@ -148,10 +148,11 @@ static void clusterSlotStatsUpdateNetworkBytesOutForReplication(long long len) {
     client *c = server.current_client;
     if (c == NULL || !canAddNetworkBytesOut(c)) return;
 
+    uint64_t rlen = len*listLength(server.replicas);
```

Comment:
```suggestion
    int64_t rlen = len * listLength(server.replicas);
```

This was confusing to me too. :) I think just fixing this makes the code easy to follow.